python/pyspark/ml/tests/test_algorithms.py

```diff
@@ -115,6 +115,7 @@ def test_output_columns(self):
         model = ovr.fit(df)
         output = model.transform(df)
         self.assertEqual(output.columns, ["label", "features", "rawPrediction", "prediction"])
+        self.assertIsInstance(output.schema["rawPrediction"].dataType, VectorUDT)
```

Comment:
Let's add a test of running "model.transform(df).head()", ensure it do not raise error.

Reply:
add a separate unit test with name "fix SPARK-35142:..." will be better.

Reply:
Got it!